andela/ah-frontend-prime
# #164798255 Enable users to view and edit their profiles

#### What does this PR do?
- Enable authenticated users to view and edit their profiles
#### Description of tasks to be completed.
- Add tests for profile and edit profile functionality
- Add functionality for getting a user's profile
- Add functionality for a user to edit their profile
#### How should this be manually tested?
- Visit the deployed application page
- Login using the registered email and password using the sample credentials below:
`email: ianemma70@gmail.com`
`password: @bochiSupreme1`
- On successful login, a toast message shows that a successful login and then you are redirected to the home/landing page with the `profile` link the navbar.
- Visit the profile page to view the current logged user's profile
- To edit a profile, click on edit profile and edit your `full_name`, `bio` and `image`.
- On a successful edit, a toast appears and you are redirected to the profile page
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#164798255](https://www.pivotaltracker.com/story/show/164798255)

#### Screenshots(if appropriate)
- profile page
![image](https://user-images.githubusercontent.com/36891299/57984829-4a1b4780-7a68-11e9-8bbe-d26448b17e41.png)

- Edit profile page
![image](https://user-images.githubusercontent.com/36891299/57984841-67e8ac80-7a68-11e9-8dd4-ba5392d73062.png)
